components/_patterns/00-base/global/01-colors/_color-vars.scss

```diff
@@ -1,12 +1,51 @@
-// Grayscale
+// Neutrals
 $white: white;
-$near-white: #f2f2f2;
-$gray-lightest: #e5e5e5;
-$gray-lighter: #ccc;
-$gray-light: #999;
-$gray: #666;
-$gray-dark: #4c4c4c;
-$gray-darker: #333;
+$gray-xxlight: #fafafa;
+$gray-xlight: #f5f5f5;
+$gray-light: #e0e0e0;
+$gray-medium: #bdbdbd;
+$gray-dark: #616161;
+$gray-xdark: #424242;
 $black: black;
 
-// Project
+// Brand Colors
+$nypl-red-light: #e9b9a7;
+$nypl-red-regular: #d0343a;
+$nypl-red-muted-regular: #bb1d12;
+$nypl-red-dark: #97272c;
+
+$nypl-yellow-light: #f9e08e;
+$nypl-yellow-regular: #ffb81d;
+$nypl-yellow-dark: #c99212;
+
+$nypl-green-light: #c5d5a5;
+$nypl-green-regular: #799a05;
+$nypl-green-dark: #497629;
+
+$nypl-turquoise-light: #b0e3e4;
+$nypl-turquoise-regular: #00a1b0;
+$nypl-turquoise-dark: #007079;
+
+$nypl-blue-light: #b1c9ea;
+$nypl-blue-regular: #0071ce;
+$nypl-blue-dark: #004b98;
+
+$nypl-purple-light: #daa9e3;
+$nypl-purple-regular: #8a1a9c;
+$nypl-purple-dark: #673165;
+
+// Theme Colors
+$books-and-more: $nypl-red-regular;
+$research: $nypl-turquoise-regular;
```

Comment:
There's a file at `01-colors/_colors-used.scss` that's intended to hold this stuff (mapping "use-cases" to color definitions.) You don't have to use it, but I'd recommend either moving this stuff into there, or deleting that file.

I also like to prefix my color variables with `$color-` I find it helpful so that later when I'm theming the spotlight, for example, I know at a glance the difference between `$color-spotlight-heading`, `$color-spotlight-bg`, `$padding-spotlight-wrapper`, `$font-spotlight-heading` etc.

The variable `$spotlight` is vague enough that 6 months from now, or if I'm a new developer to the project, I may not be 100% sure what it's going to do. Context will probably tell me, but it's always nice to have self-documenting code.

Just to beat a dead horse 😛, which do you feel more confident you understand what's going to happen when you use these variables?
`$spotlight`
`$spotlight-padding`
`$spotlight-heading`

or

`$color-spotlight-bg`
`$padding-spotlight-wrapper`
`$font-spotlight-heading`

This is all a personal preference of course (and one reason we haven't totally overhauled all the example code in Emulsify) but it's something that has helped me as I jump between projects, and return to projects after an extended break.

Reply:
I think that prefixing the color variables with `color-` is a good one, as it grounds you in their meaning when they exist beyond their declaration file, so this is a change I'll make on this PR. 

On our project it would be bad design to have our project-sections (such as `books-and-more`) have their own specific padding, fonts, etc. beyond their accent color. Your example, while perfectly reasonable, is one that we don't want to apply to us—we would like to have a series of `$padding-` declarations that all areas of the site will use.